### PR TITLE
Added skin tone option

### LINKED ISSUE #31: skin tone
Hi, wondering if there is anyway we can change the skin tone used in the emoji through this workflow?
Thanks

### PR DESCRIPTION
**Pull request by Glennmen**
- Default no skin tone (keep same behaviour)
- Added environment variable to change skin tone (0, 1, 2 ,3, 4)
- Added `shift` modifier to copy emoji without skin tone

fixes #31 

This was my first time working on an Alfred workflow 😅 but it was a lot of fun. I am open for feedback.

First credits to https://github.com/muan/mojibar for inspiration how to add skin tone to an emoji.
Also there are a lot of emoji that should support skin tone but the `emojilib` is a little out of date it seems. Hopefully it will get updated soon.
I also didn't know if I should add skin tone to Copy code `:+1:`, I couldn't find any information if there is any spec for this or if every app implements this in their own format.

For now I used an environment variable `skin_tone`, but a good next improvement would be adding a new Alfred command to change skin tone.

### CODE REVIEW

**Comment by jsumners** [comment]
What happens if a skin tone doesn't exist? Does it fall back to the neutral tone?

**Comment by Glennmen** [comment]
@jsumners Do you mean when you set a skin tone value higher than 4? It will fallback to the neutral tone.

**Comment by jsumners** [comment]
> Also there are a lot of emoji that should support skin tone but the emojilib is a little out of date it seems.

So if a user has set that they prefer some alternative tone, and the emoji they are choosing doesn't support it, what happens? Can you paste a gif of such a search?

**Comment by Glennmen** [comment]
@jsumners That is what this line is for:

https://github.com/Glennmen/alfred-emoji/blob/2973921b00747ef706d16c838becd47811bb3db0/lib/genicons.js#L18

It will check if the emoji has skin tone support.
I am also writing some extra tests right now, will commit it in a few minutes and see if I can make a video.

**Comment by Glennmen** [comment]
@jsumners Here is the gif, I hope this answers your question 😄 
emoji skin tone

Currently `emojilib` is very outdated it seems, that is why those smiles currently don't have a skin tone.

**Comment by jsumners** [comment]
Thank you for the gif; it answers my question clearly.

> Currently emojilib is very outdated it seems, that is why those smiles currently don't have a skin tone.

I know. They are working on a restructuring on the library to follow Unicode releases. It's a difficult problem to tackle. I used up a whole day on it a while back and got basically nowhere (https://twitter.com/jsumners79/status/1211105577188970497). I may get back to it soon 🤷‍♂ .

File: Readme.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/jsumners/alfred-emoji.svg?branch=master)](https://travis-ci.org/jsumners/alfred-emoji)
 
 An [Alfred workflow][alfred] that makes it easy to search for emoji and copy
-them to the clipboard 🤘.
+them to the clipboard 🤘. **Now supports skin tones!**
```
Comment (jsumners) [inline Readme.md:6]:
```suggestion
them to the clipboard 🤘.
```

File: Readme.md [hunk 2]
```diff
@@ -26,9 +26,28 @@ your clipboard.
 Press <kbd>alt</kbd>+<kbd>return</kbd> (⌥↵): **Copy the code** of the selected emoji)
 (e.g. `:rofl:`) to your clipboard.
 
+Press <kbd>shift</kbd>+<kbd>return</kbd> (⇧↵): **Copy the default symbol** of the selected emoji)
+(e.g. 🤣) to your clipboard with no skin tone modifier.
```
Comment (jsumners) [inline Readme.md:30]:
```suggestion
(e.g. 🤣) to your clipboard without skin tone modifier.
```

File: Readme.md [hunk 3]
```diff
@@ -26,9 +26,28 @@ your clipboard.
...
+
 Press <kbd>cmd</kbd>+<kbd>return</kbd> (⌘↵): **Paste the symbol** of the selected
 emoji (e.g. 🤣) directly to your frontmost application.
 
+### Set skin tone
+
+To change the emoji skin tone for supported emoji you can change the `skin_tone` variable in Alfred.
```
Comment (jsumners) [inline Readme.md:37]:
```suggestion
To change the emoji skin tone for supported emoji set the `skin_tone` environment variable in Alfred:
```

**Comment by Glennmen** [comment]
@jsumners If emojilib does the major update, it seems it will have a lot of breaking changes, feel free to tag me and if I have the time I will be happy to implement them. I went through the emojilib PR and [unicode-emoji-json](https://github.com/muan/unicode-emoji-json) yesterday and it seems doable 🤔 

I applied your suggested changed 🎉

**Comment by Glennmen** [comment]
There seems to be an error in one of the travis builds that are not related to the changes 🤔

**Comment by Glennmen** [comment]
@jsumners Anything you want me to change? Just let me know 😄

**Comment by jsumners** [comment]
No. I just need to get around to dealing with it. I want to add in some GitHub Actions (maybe).

**Pull request merged**